src/components/hv-picker-field/index.js

```diff
@@ -52,23 +52,14 @@ export default class HvPickerField extends PureComponent<
   static localNameAliases = [];
   props: HvComponentProps;
   state: State;
+  element: Element;
 
   constructor(props: HvComponentProps) {
     super(props);
-    const element: Element = props.element;
-    const value: ?DOMString = element.getAttribute('value');
+    this.element = this.props.element.cloneNode(true);
```

Comment:
I'm a bit worried about storing `element` as an instance variable, rather than always referring to `props.element`.

We want this React component to re-render itself based on new props. For example, if some user action results in a change to the `<picker-field>` XML elemen, (for example, `set-value` behavior), then we want this component to use that new element to render. If we store a reference to the first element, then we won't use newer elements for updates.

If my understanding is correct, then the "set-value" example in the Hyperview demo app will not work with the picker field.

Reply:
the reason I created it as a class attr is because I didn't want to do `this.props.element = newElement`. would getDerivedStateFromProps help me here?

Reply:
I saw the set-value example and it does not work